src/signatureUtils.js

```diff
@@ -23,23 +24,28 @@ const logger = require('@adobe/aio-lib-core-logging')(loggerNamespace,
  * @param {*} signatureOptions map of all digital signature header values consisting fields as below
  * digiSignature1 : Value of digital signature retrieved from the x-adobe-digital-signature1 header in each POST request to webhook
  * digiSignature2 : Value of digital signature retrieved from the x-adobe-digital-signature2 header in each POST request to webhook
- * publicKeyUrl1 : Value of public key url retrieved from the x-adobe-public-key1-url header in each POST request to webhook
- * publicKeyUrl2 : Value of public key url retrieved from the x-adobe-public-key2-url header in each POST request to webhook
+ * publicKeyPath1 : Relative path of ioevents public key retrieved from the x-adobe-public-key1-path header in each POST request to webhook
+ * publicKeyPath2 : Relative path of ioevents public key retrieved from the x-adobe-public-key2-path header in each POST request to webhook
  * @param {*} recipientClientId - target recipient client id
  * @param {*} signedPayload - I/O Events proper signed payload
  * @returns {boolean} true if either signatures are valid or false
  */
 async function verifyDigitalSignature (signatureOptions, recipientClientId, signedPayload) {
   const signatures = [signatureOptions.digiSignature1, signatureOptions.digiSignature2]
-  const keys = await fetchPemEncodedPublicKeys(signatureOptions.publicKeyUrl1, signatureOptions.publicKeyUrl2)
+  // complete public key url is the concatenation of the fixed adobe ioevents domain and the relative path of key
+  // url format for prod is - https://adobeioevents.com/prod/keys/pub-key-<random-uuid>.pem
```

Comment:
We can simply say: Example URL format instead of specifying it is for prod.
```suggestion
  // Example url format - https://adobeioevents.com/prod/keys/pub-key-<random-uuid>.pem
```